core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/SamDAO.scala

```diff
@@ -33,67 +34,58 @@ import scala.concurrent.Future
 trait SamDAO {
   val errorReportSource = ErrorReportSource("sam")
 
-  def registerUser(userInfo: UserInfo): Future[Option[RawlsUser]]
+  def registerUser(ctx: RawlsRequestContext): Future[Option[RawlsUser]]
 
-  def getUserStatus(userInfo: UserInfo): Future[Option[SamUserStatusResponse]]
+  def getUserStatus(ctx: RawlsRequestContext): Future[Option[SamUserStatusResponse]]
 
-  def getUserIdInfo(userEmail: String, userInfo: UserInfo): Future[SamDAO.GetUserIdInfoResult]
+  def getUserIdInfo(userEmail: String, ctx: RawlsRequestContext): Future[SamDAO.GetUserIdInfoResult]
 
-  def getProxyGroup(userInfo: UserInfo, targetUserEmail: WorkbenchEmail): Future[WorkbenchEmail]
-
-  def createResource(resourceTypeName: SamResourceTypeName, resourceId: String, userInfo: UserInfo): Future[Unit]
+  def createResource(resourceTypeName: SamResourceTypeName, resourceId: String, ctx: RawlsRequestContext): Future[Unit]
 
   def createResourceFull(resourceTypeName: SamResourceTypeName,
                          resourceId: String,
                          policies: Map[SamResourcePolicyName, SamPolicy],
                          authDomain: Set[String],
-                         userInfo: UserInfo,
+                         ctx: RawlsRequestContext,
                          parent: Option[SamFullyQualifiedResourceId]
   ): Future[SamCreateResourceResponse]
 
-  def deleteResource(resourceTypeName: SamResourceTypeName, resourceId: String, userInfo: UserInfo): Future[Unit]
+  def deleteResource(resourceTypeName: SamResourceTypeName, resourceId: String, ctx: RawlsRequestContext): Future[Unit]
 
   def userHasAction(resourceTypeName: SamResourceTypeName,
                     resourceId: String,
                     action: SamResourceAction,
-                    userInfo: UserInfo
+                    cts: RawlsRequestContext
```

Comment:
it's absolutely essential that you fix this typo: `cts` -> `ctx`

Reply:
just making sure you were paying attention